ecommerce/products/models.py

```diff
@@ -5,13 +5,31 @@
 from products import enums
 
 
+class Category(BaseAbstractModel):
+    name = models.CharField(max_length=255, verbose_name=_('Name'))
+    # ı need to create a slug field
+    # because if user wants to go a product with computer category
+    # then our url can be such as '/computer/product_name or product_id'
+    # this computer is our slug and it is unique
+    slug = models.SlugField(max_length=50, unique=True, verbose_name=_('Slug'))
```

Comment:
You could override save method and generate slug before saving.
https://www.geeksforgeeks.org/overriding-the-save-method-django-models/

Reply:
@eibrahimarisoy thank you for your advise, you can check my last changes  [here](https://github.com/upy/bootcamp/pull/51/commits/665f64e84db08cf2f6f12f1d2f0670d302f94430) .